apps/token-manager/app/src/components/InfoBoxes.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Box,
+  GU,
+  Text,
+  TokenBadge,
+  breakpoint,
+  textStyle,
+  theme,
+  useTheme,
+} from '@aragon/ui'
+import { useNetwork } from '@aragon/api-react'
+import { formatBalance, stakesPercentages } from '../utils'
+import You from './You'
+import LocalIdentityBadge from './LocalIdentityBadge/LocalIdentityBadge'
+
+const DISTRIBUTION_ITEMS_MAX = 7
+
+const displayedStakes = (accounts, total, colors) => {
+  return stakesPercentages(accounts.map(({ balance }) => balance), {
+    total,
+    maxIncluded: DISTRIBUTION_ITEMS_MAX,
+  }).map((stake, index) => ({
+    name: stake.index === -1 ? 'Rest' : accounts[index].address,
+    stake: stake.percentage,
+    color: colors[index % colors.length],
+  }))
+}
+
+class InfoBoxes extends React.PureComponent {
+  static defaultProps = {
+    holders: [],
+  }
+  transferableLabel() {
+    const { tokenTransfersEnabled } = this.props
+    if (tokenTransfersEnabled === undefined) {
+      return 'Unknown'
+    }
+    return tokenTransfersEnabled ? 'Yes' : 'No'
+  }
+  render() {
+    const {
+      holders,
+      network,
+      tokenAddress,
+      tokenDecimalsBase,
+      tokenName,
+      tokenSupply,
+      tokenSymbol,
+      userAccount,
+      theme,
+      ...rest
```

Comment:
You were probably using this in `dev`, not needed anymore.

Reply:
It’s coming from the previous version and passed to the wrapping component, but it doesn’t seem like it was ever being used indeed :+1: